package.json

```diff
@@ -6,6 +6,7 @@
     "start": "node node_modules/react-native/local-cli/cli.js start"
   },
   "dependencies": {
+    "@tradle/react-native-http": "^2.0.0",
```

Comment:
What is that?

Reply:
A scoped packaged, should not have been there, removing